docs/src/content/docs/advanced-workflows/codeowners-enforcement.mdx

```diff
@@ -0,0 +1,74 @@
+---
+title: Enforcing CODEOWNERS Approval
```

Comment:
I don't know if this changes how you want the docs at all, but really this functionality applies to any requested reviews.  It just happens that CODEOWNERS is a common way that people get reviews requested.

Reply:
Yeah that makes sense. I want to leave it for now. Might revisit.